aspnetcore/fundamentals/url-rewriting.md

```diff
@@ -16,76 +16,92 @@ uid: fundamentals/url-rewriting
 
 By [Luke Latham](https://github.com/GuardRex) and [Mikael Mengistu](https://github.com/mikaelm12)
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/url-rewriting/sample)
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/url-rewriting/samples/)
 
 URL rewriting is the act of modifying request URLs based on one or more predefined rules. URL rewriting creates an abstraction between resource locations and their addresses so that the locations and addresses are not tightly linked. There are several scenarios where URL rewriting is valuable:
 * Moving or replacing server resources temporarily or permanently while maintaining stable locators for those resources
-* Splitting request processing across different applications or across areas of one application
+* Splitting request processing across different apps or across areas of one app
 * Removing, adding, or reorganizing URL segments on incoming requests
 * Optimizing public URLs for Search Engine Optimization (SEO)
 * Permitting the use of friendly public URLs to help people predict the content they will find by following a link
 * Redirecting insecure requests to secure endpoints
 * Preventing image hotlinking
 
-You can define rules for changing the URL in several ways, including regular expression (regex) matching rules, rules based on the Apache mod_rewrite module, rules based on the IIS Rewrite Module, and with your own method and class rule logic. This document introduces URL rewriting with instructions on how to use URL Rewriting Middleware in ASP.NET Core applications.
+You can define rules for changing the URL in several ways, including regular expression (regex) matching rules, rules based on the Apache mod_rewrite module, rules based on the IIS Rewrite Module, and with your own method and class rule logic. This document introduces URL rewriting with instructions on how to use URL Rewriting Middleware in ASP.NET Core apps.
```

Comment:
TL4MT (Too long for machine translation)
Rules for changing a URL can be defined in several ways:

* regex